tests/Aspire.Hosting.Tests/Aspire.Hosting.Tests.csproj

```diff
@@ -8,11 +8,18 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\src\Aspire.Hosting\Aspire.Hosting.csproj" />
-    <ProjectReference Include="..\..\src\Aspire.Hosting.Azure\Aspire.Hosting.Azure.csproj" />
-    <ProjectReference Include="..\testproject\TestProject.AppHost\TestProject.AppHost.csproj" />
-    
+    <ProjectReference Include="..\..\src\Aspire.Hosting\Aspire.Hosting.csproj" AppHostLibrary="true" />
```

Comment:
Why is the test project configured with `IsAspireHost=true`, thus necessitating all the `AppHostLibrary` overrides?

Reply:
IsAspireHost was added in https://github.com/dotnet/aspire/pull/560. I assume it is because the Hosting tests create DistributedApplicationBuilder instances inline in the tests. @davidfowl ?

Reply:
Correct.